src/tasks/__tests__/processBulkUpload.int.test.ts

```diff
@@ -53,19 +55,30 @@ const createTestBulkUpload = async (
 	return bulkUpload;
 };
 
-const createTestBaseFields = async (): Promise<void> => {
-	await db.sql<BulkUpload>('baseFields.insertOne', {
-		label: 'Proposal Submitter Email',
-		description: 'The email address of the person who submitted the proposal.',
-		shortCode: 'proposal_submitter_email',
-		dataType: 'string',
-	});
-	await db.sql<BulkUpload>('baseFields.insertOne', {
-		label: 'Organization Name',
-		description: 'The name of the applying organization.',
-		shortCode: 'organization_name',
-		dataType: 'string',
-	});
+const createTestBaseFields = async (): Promise<BaseField[]> => {
+	const proposalSubmitterEmailBaseFieldResult = await db.sql<BaseField>(
+		'baseFields.insertOne',
+		{
+			label: 'Proposal Submitter Email',
+			description:
+				'The email address of the person who submitted the proposal.',
+			shortCode: 'proposal_submitter_email',
+			dataType: 'string',
+		},
+	);
+	const organizationNameBaseFieldResult = await db.sql<BaseField>(
+		'baseFields.insertOne',
+		{
+			label: 'Organization Name',
+			description: 'The name of the applying organization.',
+			shortCode: 'organization_name',
+			dataType: 'string',
+		},
+	);
+	return [
+		proposalSubmitterEmailBaseFieldResult.rows[0],
```

Comment:
Surprised TypeScript lets you get away with this! (I say, knowing it doesn't because you just told me.)

Reply:
Turns out it actually wasn't angry there -- it was angry later when I tried to *use* the return values.

I fixed it to guarantee that exactly two values are returned (which, of course, does bump up the check requirement to taking place here)